addons/web/static/tests/views/fields/datetime_field.test.js

```diff
@@ -497,3 +497,57 @@ test("datetime field: use picker with arabic numbering system", async () => {
     await animationFrame();
     expect(queryFirst("[name=datetime] input")).toHaveValue("٠٢/٠٨/٢٠١٧ ١١:٤٥:٠٠");
 });
+
+test("datetime field in list view with showSeconds option", async () => {
+    mockTimeZone(+2);
+    onRpc("has_group", () => true);
+
+    await mountView({
+        type: "list",
+        resModel: "partner",
+        arch: /* xml */ `
+            <tree>
+                <field name="datetime" widget="datetime" options="{'showSeconds': false}" string="showSeconds as false"/>
+                <field name="datetime" widget="datetime" options="{'showSeconds': true}" string="showSeconds as true"/>
+            </tree>`,
+    });
+
+    const [dateField1, dateField2] = queryAll(".d-flex.gap-2.align-items-center");
+    const expectedDateString1 = "02/08/2017 12:00";
+    const expectedDateString2 = "02/08/2017 12:00:00";
+
+    expect(dateField1).toHaveText(expectedDateString1, {
+        message: "seconds should be hidden for showSeconds false",
+    });
+
+    expect(dateField2).toHaveText(expectedDateString2, {
+        message: "seconds should be visible for showSeconds true",
+    });
+});
+
+test("datetime field in form view with showSeconds option", async () => {
+    mockTimeZone(+2);
+
+    await mountView({
+        type: "form",
+        resModel: "partner",
+        arch: /* xml */ `
+            <form>
+                <field name="datetime" widget="datetime" options="{'showSeconds': false}" string="showSeconds as false"/>
+                <field name="datetime" widget="datetime" options="{'showSeconds': true}" string="showSeconds as true"/>
```

Comment:
remove this one, not relevant here

Reply:
@Polymorphe57 I am not fully understanding what is not relevant here. Is it the `show_seconds` property set to true, which is true by default, or is it something else? Could you please help clarify?